### ISSUE TITLE
[Keyboard Navigation-Howdyai]: Focus indicator is not visible on “Pause” button present in the howdy page. 

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/howdy-website#4]
**User Experience**
Users who depend on Keyboard navigation will be getting affected, if the focus indicator is not visible on “Pause” button then the keyboard user will not able to identified that control. .
 
**Test Environment**
OS: Win 10 Version 1809 (OS Build 17763.379)
URL: https://howdy.ai/
Browser: Edge

**Repro Steps**
Steps to Follow Install Slack:
Step 1: Open the URL: https://howdy.ai/
Step 2: Navigate to "@howdy for Slack" and select it.
Step 3: Navigate to "Create Workspace" and select it.
Step 4: Sign In with valid credentials to open slack account.
Step 5: Now go to Howdy Home page URL: https://howdy.ai/
Step 6: Repeat Step 2 again and select it.
Step 7: Navigate to "Install" button and select it.
Step 8: Select "Authorize" button and select it.
Step 9: Select "Turn your Boot On"
Step 10: Click on "Continue" button.
Step 11: "Howdy Dashboard" page gets opened.
Step 12: Navigate to "Launch Howdy in Slack".
Step 13: Navigate to "Dashboard" tab and select it.
Step 14: "Howdy Dashboard" page gets opened.
Step 15: Navigate to Pause button using tab key
Step 16: Observe the issue.

**Actual Result**
Focus indicator is not visible on “Pause” button present in the howdy page. 

**Expected Result** 
Focus indicator should be visible on “Pause” button present in the howdy page. 

**Mas Reference**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c
[MAS2.4.7_Pause all.zip](https://github.com/howdyai/howdy-website/files/2973533/MAS2.4.7_Pause.all.zip)

**Comment by v-gasikk** [comment]
Verified. Issue is Fixed.

Now Focus indicator is visible on "Pause All" button and "Unpause All" button.
